google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Subscriber.java

```diff
@@ -341,10 +336,9 @@ private void startStreamingConnections() throws IOException {
                 ackExpirationPadding,
                 maxAckExtensionPeriod,
                 ackLatencyDistribution,
-                subStub,
+                GrpcSubscriberStub.create(subStubSettings),
```

Comment:
is it okay to lose the IllegalStateException?

Reply:
That only existed because the AbstractApiService doStart can't throw a non-RuntimeException.  startStreamingConnections already throws an IOException.